extensions/wikia/Wall/js/Wall.js

```diff
@@ -891,6 +901,25 @@
 		classBindings: {}
 	};
 
+	// Static methods and properties
+	/**
+	 * Whether we are using Monobook
+	 */
+	Wall.isMonobook = (mw.config.get('skin') === 'monobook');
+
+	/**
+	 * SUS-550: Show a login popup message for anons on Monobook
+	 */
+	Wall.showMonobookLoginPopup = function () {
+		new BannerNotification(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)